src/Controller/AppController.php

```diff
@@ -26,11 +27,26 @@ class AppController extends Controller {
 			'secure' => true
 		],*/
 		'Auth' => [
+			'className' => 'AclAuth',
+			'allowedActionsForBanned' => [
+				'Pages' => [
+					'home'
+				]
+			],
 			'authenticate' => [
 				'Form',
 				'Xety/Cake3CookieAuth.Cookie'
 			],
-			'authorize' => ['Controller'],
+			'flash' => [
+				'element' => 'error',
+				'key' => 'flash',
+				'params' => ['class' => 'error']
+			],
+			'authorize' => [
+				'Acl.Actions' => [
+					'actionPath' => 'app/'
```

Comment:
Alors en faite, le `actionPath` représente tout simplement la racine du site, et tout les controllers ont en parent cet aco, avec une image tu va mieux comprendre :
![screenshot_7](https://cloud.githubusercontent.com/assets/8210023/5918353/183fc1fa-a628-11e4-841f-fad6f67820e2.png)
![screenshot_8](https://cloud.githubusercontent.com/assets/8210023/5918365/2c99888e-a628-11e4-9a87-48d2b0f20b19.png)

Ce qui permet de faire des permissions sans trop ce prendre la tête, car par exemple, si tu veux autoriser le groupe `1` à accéder juste aux actions du `[4] Blog` et du controller `[19] Pages` , tu peux faire ceci :
```php
$group = $this->Groups->get(1);

$this->Acl->deny($group, 'app');
$this->Acl->allow($group, 'app/Pages');
$this->Acl->allow($group, 'app/Blog');
```

Alors que sans cette `racine`, on aurait fait un `deny()` sur chaque controller et aussi sur le prefix `[44] admin` ce qui peut vite devenir chiant sur un gros site avec beaucoup de controllers.
Dans la doc, il nomme ça `controller/`, moi j'ai choisi `app/` car c'est plus court et pour faire les permissions je gagne un poil de temps à écrire, mais sinon tu peux nommer car comme bon te semble.
http://book.cakephp.org/2.0/fr/tutorials-and-examples/simple-acl-controlled-application/simple-acl-controlled-application.html#creer-les-acos-access-control-objects